lua/neo-tree/ui/renderer.lua

```diff
@@ -640,12 +640,12 @@ end
 M.position = {
   save = function(state)
     if state.tree and M.window_exists(state) then
-      local success, node = pcall(state.tree.get_node, state.tree)
-      if success and node then
-        _, state.position.node_id = pcall(node.get_id, node)
-        local win_state = vim.fn.winsaveview()
-        state.position.topline = win_state.topline
-      end
+      local win_state = vim.api.nvim_win_call(state.winid, vim.fn.winsaveview)
+      log.trace("win_state: " .. vim.inspect(win_state))
```

Comment:
Just a minor pick; I don't think it is good to leave `vim.inspect` inside the codebase, as the log.debug below already gives good info.